dashboard/proto/gen/catalog.ts

```diff
@@ -12,9 +12,9 @@ import {
 export const protobufPackage = "catalog";
 
 /**
- * The rust prost library always treats uint64 as required and message as optional.
```

Comment:
I think the changes to this file are irrelevant to this PR?

Reply:
These codes were generated.